src/storage.cc

```diff
@@ -124,8 +143,8 @@ void Storage::Tick() {
   LOG(cyclus::LEV_INFO4, "ComCnv") << "current capacity " << max_inv_size << " - " << processing.quantity() << " - " << ready.quantity() << " - " << stocks.quantity() << " = " << current_capacity();
 
   // Set available capacity for Buy Policy
-  inventory.capacity(current_capacity());
-
+  // not necessary any more
+  // inventory.capacity(current_capacity());
```

Comment:
delete comments

Reply:
also delete the dangling comment above the deleted line

Reply:
I think we'll be ready to merge (pending tests) once you delete these lines...

Reply:
I am rebasing and ran into some errors, so I'm currently dealing with those